injection/sharedmain/main.go

```diff
@@ -149,9 +153,6 @@ func MainWithConfig(ctx context.Context, component string, cfg *rest.Config, cto
 
 	ctx, informers := injection.Default.SetupInformers(ctx, cfg)
```

Comment:
This line is what sets up the `kubeclient` that is used in `SetupLoggerOrDie`, so this is going to blow up everything downstream.  I'm gonna rollback.